openmc/settings.py

```diff
@@ -207,6 +207,10 @@ class Settings:
                    banked (int)
         :max_particles: Maximum number of particles to be banked on surfaces per
                    process (int)
+        :split_file_per_batch: create surface source file per batch (bool)
+        :n_surf_source_files: Number of surfaces source files to be created 
+                   for the n last batches (int)
+        :info_surface_source: print number of particle saved in surface source file (bool)
```

Comment:
Yes, I think it's better. I decided not to include the number of particles, so that it is exactly like summary.h5 and statepoint.*.h5. Is that ok with you?

Reply:
In this case I think it _would_ he helpful for the user to know how many particles are being written each batch as it gives them an idea of how many surface crossings will be banked for the entire simulation, but I'm open to what you think is best.

Reply:
I had changed it to be exactly like the case of statepoint and summary, but you are right, it's worth to write the number of saved particles. I've added it!